extensions/resteasy-reactive/rest-client-reactive/deployment/src/main/java/io/quarkus/rest/client/reactive/deployment/RestClientReactiveProcessor.java

```diff
@@ -557,6 +492,108 @@ void addRestClientBeans(Capabilities capabilities,
         }
     }
 
+    private void populateClientExceptionMappersFromAnnotations(BuildProducer<GeneratedClassBuildItem> generatedClasses,
```

Comment:
The method name probably needs to be updated :)

Reply:
I've renamed it to `populateClientExceptionMapperFromAnnotations`. 
We have two methods:
- `populateClientExceptionMapperFromAnnotations` to populate `@ClientExceptionMapper`
- `populateClientRedirectHandlerFromAnnotations` to populate `@ClientRedirectHandler`